packages/wallet/src/components/dialogs/SuccessDialog.tsx

```diff
@@ -59,25 +75,26 @@ export const SuccessDialog = ({
                   </div>
                 </div>
                 <div className="mt-5 grid grid-cols-1 gap-3">
-                  <Button
-                    intent="success"
-                    aria-label="close dialog"
-                    fullWidth
-                    onClick={() => onClose()}
-                  >
-                    Close
-                  </Button>
                   {redirect && (
```

Comment:
Can we remove the `redirect` check now since we are handling this with `successButtonProps`? 🤔